src/userApps.ts

```diff
@@ -146,4 +162,25 @@ export class UserApps {
         if (!res.verified) throwError('JWT failed verification', SdkErrors.JwtNotValid);
         return res;
     }
+
+    static async verifyPrivateKey(accountName: string, keyManager?: KeyManager): Promise<boolean> {
+        let privateKey;
+        if (keyManager) {
+            privateKey = await keyManager.getKey({
+                level: KeyManagerLevel.LOCALSTORAGE,
+            });
+
+            privateKey = privateKey?.toString();
+        } else {
+            privateKey = localStorage.getItem(KeyManagerLevel.LOCALSTORAGE);
```

Comment:
We should not need to interface with localStorage. That should all be handled within `keyManager.getKey()`

Reply:
The principle behind keyManager is that all private keys are, once they are inside, never leave the keyManager object and that is the only place that you can access them through. This makes auditing the code base the security much easier and gives people  a clear picture of the security of application.